src/Marten/Schema/Arguments/IFunctionArgument.cs

```diff
@@ -0,0 +1,31 @@
+using NpgsqlTypes;
+
+namespace Marten.Schema.Arguments
+{
+    /// <summary>
+    /// Represents function argument.
+    /// </summary>
+    public interface IFunctionArgument
```

Comment:
**CONCERN:** @jeremydmiller, if we're open to adding that, then this should probably go to Weasel.